armips/data/moves.s

```diff
@@ -12667,7 +12667,7 @@ movedata MOVE_APPLE_ACID, "Apple Acid"
     movedescription MOVE_APPLE_ACID, "---"
 
 movedata MOVE_GRAV_APPLE, "Grav Apple"
-    battleeffect MOVE_EFFECT_HIT
+    battleeffect MOVE_EFFECT_LOWER_DEFENSE_HIT // damage portion handled in CalcBaseDamage.c
```

Comment:
is this still the case?

Reply:
yea it's like hydro steam where it's just. if condition is grav 1.5x movepower